examples/tweet_generator.py

```diff
@@ -0,0 +1,147 @@
+            box=(15, 15)
+        )
+
+        # add the fonts
+        font = ImageFont.truetype('Segoe UI.ttf', 40)
+        font_small = ImageFont.truetype('Segoe UI.ttf', 30)
+        font_bold = ImageFont.truetype('Segoe UI Bold.ttf', 40)
+
+        # write the name and username on the Image
+        draw = ImageDraw.Draw(tweet)
+        draw.text((180, 20), str(ctx.author.user), fill=(0, 0, 0),
+                  font=font_bold)
+        draw.text((180, 70), '@' + ctx.author.user.username, fill=(120, 120, 120),
+                  font=font)
+
+        # write the content of the tweet on the Image
+        message = '\n'.join(message).split(' ')
+        result = []
+
+        # generate a dict to set were the text need to be in different color.
+        # for example, if a word starts with '@' it will be write in blue.
+        # example:
+        #   [
+        #       {'color': (0, 0, 0), 'text': 'hello world '},
+        #       {'color': (0, 154, 234), 'text': '@drawbu'}
+        #   ]
+        for word in message:
+            for i_o, o in enumerate(word.split('\n')):
+
+                o += '\n' if i_o != len(word.split('\n')) - 1 else ' '
+
+                if not result:
+                    result.append({'color': (0, 0, 0), 'text': o})
+                    continue
+
+                if not o.startswith('@'):
+                    if result[-1:][0]['color'] == (0, 0, 0):
+                        result[-1:][0]['text'] += o
+                        continue
+
+                    result.append({'color': (0, 0, 0), 'text': o})
+                    continue
+
+                result.append({'color': (0, 154, 234), 'text': o})
+
+        # write the text
+        draw = ImageDraw.Draw(tweet)
+        x = 30
+        y = 170
+        for o in result:
+            y -= font.getsize(' ')[1]
+            for l_index, line in enumerate(o['text'].split('\n')):
+                if l_index != 0:
+                    x = 30
+                y += font.getsize(' ')[1]
+                draw.text((x, y), line, fill=o['color'], font=font)
+                x += font.getsize(line)[0]
+
+        # write the footer
+        draw.text(
+            (30, tweet.size[1] - 60),
+            datetime.now().strftime(
+                '%I:%M %p · %d %b. %Y · Twitter for Discord'),
+            fill=(120, 120, 120),
+            font=font_small)
+
+        return Message(
+            embeds=[
+                Embed(
+                    title='Twitter for Discord',
+                    description=''
+                ).set_image(url="attachment://image0.png")
+            ],
+            attachments=[tweet]
+        )
+
+
+# https://stackoverflow.com/a/53663233/15485584
+def trans_paste(fg_img, bg_img, alpha=1.0, box=(0, 0)):
+    """
+    paste an image into one another
+    """
+    fg_img_trans = Image.new("RGBA", fg_img.size)
+    fg_img_trans = Image.blend(fg_img_trans, fg_img, alpha)
+    bg_img.paste(fg_img_trans, box, fg_img_trans)
+    return bg_img
```

Comment:
Imo you should make box a regular argument and remove alpha from the arguments since its unused.